vmm/src/lib.rs

```diff
@@ -455,25 +456,27 @@ pub fn start_vmm_thread(
         None => None,
     };
 
-    if let Some(http_path) = http_path {
-        api::start_http_path_thread(
+    let http_thread = if let Some(http_path) = http_path {
```

Comment:
nit: I would have named this `http_api_thread_handle` so you don't need to alias it when you pass it to the function below.

Reply:
Fixed, I changed to http_api_handle.